.github/workflows/index.yml

```diff
@@ -19,10 +17,34 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Install dependencies
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          version: 7
+          run_install: false
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        shell: bash
         run: |
-          npm install -g pnpm
-          pnpm install
+          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_ENV
```

Reply:
$GITHUB_OUTPUT will be used by the next step for Setup pnpm cache

path input specifies the directory path to cache, which is the output from a previous step in the workflow called pnpm-cache which is outputed in $GITHUB_OUTPUT, pnpm will not work with $GITHUB_ENV

[I used this documentation as reference](https://github.com/marketplace/actions/setup-pnpm)